__tests__/endpoint.tests.js

```diff
@@ -142,5 +142,52 @@ describe('GET/api/articles/:article_id/comments', () => {
         const err = result.body;
         expect(err.msg).toBe('No entry for id 888');
     })
+})
+
+describe('POST /api/articles/:article_id/comments', () => {
+    it('200: returns posted object with provided body and votes(set to 0) + created_at + comment_id added', async () => {
+        const body = {
+            userName: "rogersop",
+            body: "I really liked this comment here."
+        }
+        const result = await request(app).post('/api/articles/2/comments').send(body)
+        
+        // calculate time difference between date now and db created_at time to allow for lag. Have allowed 5ms lag in expect statement below.
+        const now = new Date();
```

Comment:
I see the thought process behind this but not sure it's necessary - we just want to make sure we have a created_at property but an intuitive way to test the value!